src/main/java/com/example/petbuddybackend/dto/address/AddressDTO.java

```diff
@@ -1,14 +1,17 @@
 package com.example.petbuddybackend.dto.address;
 
 import com.example.petbuddybackend.entity.address.Voivodeship;
+import jakarta.validation.constraints.NotNull;
+import lombok.Builder;
 
+@Builder
 public record AddressDTO(
     Long id,
-    String city,
-    String zipCode,
-    Voivodeship voivodeship,
-    String street,
-    String buildingNumber,
+    @NotNull String city,
+    @NotNull String zipCode,
+    @NotNull Voivodeship voivodeship,
+    @NotNull String street,
+    @NotNull String buildingNumber,
```

Comment:
Może dać NotBlank zamiast NotNull dla city, zipCode i street?

Reply:
Ale wtedy jak coś to ta metoda serwisu z PATCH też by była do zmiany

Reply:
aa rzeczywiscie